<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LessonEdit</class>
 <widget class="QWidget" name="LessonEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="informationGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLineEdit" name="branchNumberEdit">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="creditUnitEdit">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>nameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="totalCapacityEdit">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="branchNumberLabel">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Branch Number: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>branchNumberEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="finalExamEdit">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="finalExamLabel">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Final Exam: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>finalExamEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="creditUnitLabel">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Credit Unit: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>creditUnitEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="totalCapacityLabel">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Total Capacity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>totalCapacityEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="resetNameButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/root/dialogs/reset.ico</normaloff>:/root/dialogs/reset.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="resetCreditUnitButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/root/dialogs/reset.ico</normaloff>:/root/dialogs/reset.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="resetFinalExamButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/root/dialogs/reset.ico</normaloff>:/root/dialogs/reset.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="resetBranchNumberButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/root/dialogs/reset.ico</normaloff>:/root/dialogs/reset.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="resetTotalCapacityButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/root/dialogs/reset.ico</normaloff>:/root/dialogs/reset.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="identifierGroup">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Identifier</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="entity" native="true">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>creditUnitEdit</tabstop>
  <tabstop>finalExamEdit</tabstop>
  <tabstop>branchNumberEdit</tabstop>
  <tabstop>totalCapacityEdit</tabstop>
  <tabstop>resetNameButton</tabstop>
  <tabstop>resetCreditUnitButton</tabstop>
  <tabstop>resetFinalExamButton</tabstop>
  <tabstop>resetBranchNumberButton</tabstop>
  <tabstop>resetTotalCapacityButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>resetNameButton</sender>
   <signal>clicked()</signal>
   <receiver>LessonEdit</receiver>
   <slot>resetName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetCreditUnitButton</sender>
   <signal>clicked()</signal>
   <receiver>LessonEdit</receiver>
   <slot>resetCreditUnit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetFinalExamButton</sender>
   <signal>clicked()</signal>
   <receiver>LessonEdit</receiver>
   <slot>resetFinalExam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetBranchNumberButton</sender>
   <signal>clicked()</signal>
   <receiver>LessonEdit</receiver>
   <slot>resetBranchNumber()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetTotalCapacityButton</sender>
   <signal>clicked()</signal>
   <receiver>LessonEdit</receiver>
   <slot>resetTotalCapacity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetName()</slot>
  <slot>resetFinalExam()</slot>
  <slot>resetBranchNumber()</slot>
  <slot>resetTotalCapacity()</slot>
  <slot>resetCreditUnit()</slot>
  <slot>setRandomIdentifier(Qt::CheckState)</slot>
 </slots>
</ui>
